src/cryptoadvance/specter/templates/base.jinja

```diff
@@ -40,6 +40,7 @@
 			}
 		</style>
 		<script type="text/javascript" src="{{ url_for('static', filename='helpers.js') }}"></script>
+		{% include "includes/js-helpers.jinja" %}
```

Comment:
I guess you didn't put the JS code from` js-helpes.jinja` in the already exisitng` helpers.js `because you wanted to also use jinja? The naming is a bit confusing to me. Perhaps just` helpers.jinja`?